requirements.txt

```diff
@@ -5,7 +5,7 @@ topiary>=0.1.0, <0.2.0
 mhctools>=0.3.0, <0.4.0
 varcode>=0.5.15, <0.6.0
 pyensembl>=1.0.1, <1.1.0
-lifelines>=0.9.1.0
+lifelines>=0.9.3.2
```

Comment:
Just OOC: is there a particular reason as to why we are way behind their release cycle? Looks like their most recent release is 0.11.1 which actually seems to address a similar issue: https://github.com/CamDavidsonPilon/lifelines/issues/304